kubectl-check.tf

```diff
@@ -29,6 +29,8 @@ resource "null_resource" "update_kubeconfig_with_cluster_info" {
         update-kubeconfig \
         --name ${aws_eks_cluster.cluster.name}
     EOT
+
+    interpreter = ["bash", "-c"]
```

Comment:
I don't think this is a good idea. Why do we need it?

Reply:
I saw it on the other `tf` files and I thought on homogenizing this file too.